src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackPageUiTest.java

```diff
@@ -488,10 +488,12 @@ public void testCopyToAction() throws Exception {
         
         ______TS("Submit empty course list: Feedbacks Page");
         
-        feedbackPage.clickFsCopyButton(courseId, feedbackSessionName);
-        feedbackPage.waitForModalToLoad();
-        feedbackPage.clickFsCopySubmitButton();
-        feedbackPage.verifyStatus(Const.StatusMessages.FEEDBACK_SESSION_COPY_NONESELECTED);
+        feedbackPage.fsCopyToModal.clickCopyButtonOnTable(courseId, feedbackSessionName);
+        feedbackPage.fsCopyToModal.waitForModalToLoad();
+        feedbackPage.fsCopyToModal.clickSubmitButton();
+        feedbackPage.fsCopyToModal.waitForFormSubmissionStatusMessageVisibility();
+        assertTrue(feedbackPage.fsCopyToModal.isFormSubmissionStatusMessageVisible());
```

Comment:
Is it necessary to assert that the message is visible after waiting for it?
I can understand if it's just once to ensure the method works, but multiple times seem excessive.